src/code_reviewer/lg_server.py

```
@@ -0,0 +1,10 @@
+from code_reviewer.graph import non_github_code_reviewer_config
```

Ruff (0.8.2)

Missing docstring in public module

(D100)